Gems/ROS2/Code/Source/VehicleDynamics/VehicleModelComponent.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "ManualControlEventHandler.h"
+#include "ROS2/VehicleDynamics/VehicleInputControlBus.h"
```

Comment:
This include should use angle brackets.
```suggestion
#include <ROS2/VehicleDynamics/VehicleInputControlBus.h>
```